src/assets/scss/variables/_color.variables.module.scss

```diff
@@ -57,6 +57,9 @@ $tb-green-success: #00c083;
 // text
 $tb-black-text: #000000;
 
+// logo
+$tb-logo-1: #24F3D2;
```

Comment:
Picky: I don't find the `logo-1` a meaningful name, maybe `logo-green` or `green-logo`, with something like this we will know that is a green color variant

Reply:
I put that name because it was named like that on figma, but I can change it. On the other hand, I wasn't sure if it is green or light blue

Reply:
>  I wasn't sure if it is green or light blue

I also can't be 100% sure about that hahaha